core/src/test/scala/io/delta/tables/DeltaTableSuite.scala

```diff
@@ -168,6 +168,74 @@ class DeltaTableSuite extends QueryTest
     }
   }
 
+  test("isDeltaTableByPath - with _delta_log dir") {
+    withTempDir { dir =>
+      testData.write.format("delta").save(dir.getAbsolutePath)
+      assert(DeltaTable.isDeltaTableByPath(dir.getAbsolutePath))
+    }
+  }
+
+  test("isDeltaTableByPath - with empty _delta_log dir") {
+    withTempDir { dir =>
+      new File(dir, "_delta_log").mkdirs()
+      assert(!DeltaTable.isDeltaTableByPath(dir.getAbsolutePath))
+    }
+  }
+
+  test("isDeltaTableByPath - with no _delta_log dir") {
+    withTempDir { dir =>
+      assert(!DeltaTable.isDeltaTableByPath(dir.getAbsolutePath))
+    }
+  }
+
+  test("isDeltaTableByPath - with non-existent dir") {
+    withTempDir { dir =>
+      JavaUtils.deleteRecursively(dir)
+      assert(!DeltaTable.isDeltaTableByPath(dir.getAbsolutePath))
+    }
+  }
+
+  test("isDeltaTableByPath - with non-Delta table path") {
+    withTempDir { dir =>
+      testData.write.format("parquet").mode("overwrite").save(dir.getAbsolutePath)
+      assert(!DeltaTable.isDeltaTable(dir.getAbsolutePath))
+    }
+  }
```

Comment:
These are the same tests as we use on the deprecated API--can we combine them?

Reply:
@allisonport-db  Shall I just deprecate these and have them delegate to the corresponding new ones?

Reply:
we should keep the existing tests, deprecated doesn't mean we can break it

Reply:
So what I meant is since these are duplicates what we can do is for each test do something like this:

```
  test("isDeltaTable - path - with _delta_log dir") {
    withTempDir { dir =>
      testData.write.format("delta").save(dir.getAbsolutePath)
      assert(DeltaTable.isDeltaTable(dir.getAbsolutePath))
      assert(DeltaTable.isDeltaTableByPath(dir.getAbsolutePath))
    }
  }
```
This way we don't have duplicate tests for each API